### ISSUE TITLE
[0.9.4.2] A tag key that is the same as a field key cannot be queried directly

### ISSUE THREAD

**Issue by gunnaraasen** [influxdata/influxdb#4630]
If points are written to a measurement with a tag key and field key that are equal, the points with the tag key are not returned in query results when the tag key is included in any `WHERE` clause conditional using the equals (`=`) or regex (`=~`) operators.

```
$ influx
Connected to http://localhost:8086 version 0.9.4.2
InfluxDB shell 0.9.4.2
> create database test
> use test
Using database test
> format json
> insert cpu,missing=tag missing="field",value=1
> select * from cpu
{"results":[{"series":[{"name":"cpu","columns":["time","missing","value"],"values":[["2015-10-31T00:29:33.041336768Z","field",1]]}]}]}
> select * from cpu where missing = 'tag'
{"results":[{}]}
> select * from cpu where missing =~ /tag/
{"results":[{}]}
> select * from cpu where missing = 'field'
{"results":[{"series":[{"name":"cpu","columns":["time","missing","value"],"values":[["2015-10-31T00:29:33.041336768Z","field",1]]}]}]}
```

This could be fixed by either addressing issue https://github.com/influxdb/influxdb/issues/2615 or adding the field key renaming feature described in https://github.com/influxdb/influxdb/issues/4156.

**Comment by jsternberg** [comment]
Closing this in favor of #6519.

**Comment by simplyviks** [comment]
can somebody help me with this issue , i am not able to get this work.?

InfluxDB version: 1.5.4

SHOW FIELD KEY


name: testcase_point
fieldKey           fieldType
--------           ---------
build_number       integer
project_name       string
project_path       string
rf_critical_failed float
rf_critical_passed float
rf_duration        float
rf_failed          float
rf_name            string
rf_passed          float
rf_suite_name      string
>



SHOW TAG KEY

name: testcase_point
tagKey
------
project_name
rf_name

so basically the filed and tag key name are same , so i am not able to get the below query work

show tag values from "testcase_point" with key in ("rf_name")  WHERE "project_name" = "Altiplano_AC_SystemTest"

actually WHERE CLAUSE is not working here it always returns the same set of series irrespective of WHERE clause.

Thanks in advance||
